mobile/src/main/java/rs/readahead/washington/mobile/media/MediaFileHandler.java

```diff
@@ -645,7 +645,7 @@ public static Observable<List<VaultFile>> getLastVaultFileFromDb() {
     public static void startImportFiles(Activity context, Boolean multipleFile, String type) {
         Intent intent = new Intent()
                 .setType(type)
-                .setAction(Intent.ACTION_GET_CONTENT)
+                .setAction(Intent.ACTION_OPEN_DOCUMENT)
                 .putExtra(Intent.EXTRA_ALLOW_MULTIPLE, multipleFile);
```

Comment:
@ahlem-jarrar  ACTION_GET_CONTENT is used if you want your app to simply read/import data. With this approach, the app imports a copy of the data, such as an image file.
ACTION_OPEN_DOCUMENT is used if you want your app to have long term, persistent access to documents owned by a document provider and offer the user the possiblity of editing files.                                                                                       And as it is only used in importing files I don't think it will impact the app, I tried to import different types (video/ img/ files ) and it works just fine